lib/streamlit/elements/multiselect.py

```diff
@@ -104,6 +104,26 @@ def _check_and_convert_to_indices(
     return [opt.index(value) for value in default_values]
 
 
+def get_default_count(default: Union[Sequence[Any], Any, None]) -> int:
```

Comment:
nit (both this helper function and the on below): Could prefix these with underscores since we don't expect them to be used outside of the multiselect widget (aside from in tests)